### ISSUE TITLE
Checkout totals order in specific store

### ISSUE THREAD

**Issue by daniel-ifrim** [magento/magento2#22380]
### Preconditions (*)

1. Magento 2.3.0
2.

### Steps to reproduce (*)

1. On website scope configuration change sort order for shipping
2. Admin > Stores > Configuration > Sales > Checkout Totals Sort Order > Shipping = 500
3. Clear cache

### Expected result (*)

1. On cart page in totals summary shipping row (titles + price) should come after Grand Total

### Actual result (*)

1. Shipping row is in the same place. It's because sort order is taken from "Default Config"

Looks like in `\Magento\Checkout\Model\Layout\AbstractTotalsProcessor::sortTotals`:
```
$configData = $this->scopeConfig->getValue('sales/totals_sort');
```
should be:
```
$configData = $this->scopeConfig->getValue('sales/totals_sort', \Magento\Store\Model\ScopeInterface::SCOPE_STORES);
```
### Additional Information (*)

You need to change store view to Main website.

**Comment by krishprakash** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @krishprakash. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @krishprakash, here is your [Magento instance](https://i-22380-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-22380-2-3-develop.instances.magento-community.engineering/admin](https://i-22380-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by krishprakash** [comment]
@daniel-ifrim. Thank you for your report.
I am not able to reproduce the issue on clean magento 2.3-develop instance.
This may be related to your custom modules.

**Comment by daniel-ifrim** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @daniel-ifrim. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @daniel-ifrim, here is your [Magento instance](https://i-22380-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-22380-2-3-develop.instances.magento-community.engineering/admin](https://i-22380-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by daniel-ifrim** [comment]
@krishprakash I made a quick test. Changed sort totals order for shipping on website (not on Default Config scope). I did flush cache in admin.
The bug is there. Would you please take a look again ?

https://imgur.com/a/PZTpwBW

From cart page:
```
{
   "#cart-totals":{
      "Magento_Ui/js/core/app":{
         "components":{
            "block-totals":{
               "component":"Magento_Checkout\/js\/view\/cart\/totals",
               "displayArea":"totals",
               "config":{
                  "template":"Magento_Checkout\/cart\/totals"
               },
               "children":{
                  "subtotal":{
                     "component":"Magento_Tax\/js\/view\/checkout\/summary\/subtotal",
                     "config":{
                        "title":"Subtotal",
                        "template":"Magento_Tax\/checkout\/summary\/subtotal",
                        "excludingTaxMessage":"(Excl. Tax)",
                        "includingTaxMessage":"(Incl. Tax)"
                     },
                     "sortOrder":"10"
                  },
                  "shipping":{
                     "component":"Magento_Tax\/js\/view\/checkout\/cart\/totals\/shipping",
                     "config":{
                        "title":"Shipping",
                        "template":"Magento_Tax\/checkout\/cart\/totals\/shipping",
                        "excludingTaxMessage":"Excl. Tax",
                        "includingTaxMessage":"Incl. Tax"
                     },
                     "sortOrder":"30"
                  },
                  "grand-total":{
                     "component":"Magento_Tax\/js\/view\/checkout\/cart\/totals\/grand-total",
                     "config":{
                        "title":"Order Total",
                        "template":"Magento_Tax\/checkout\/cart\/totals\/grand-total",
                        "exclTaxLabel":"Order Total Excl. Tax",
                        "inclTaxLabel":"Order Total Incl. Tax"
                     },
                     "sortOrder":"100"
                  },
                  "discount":{
                     "component":"Magento_SalesRule\/js\/view\/cart\/totals\/discount",
                     "config":{
                        "title":"Discount"
                     },
                     "sortOrder":"20"
                  },
                  "before_grandtotal":{
                     "component":"uiComponent",
                     **"sortOrder":"30",**
                     "children":[

                     ]
                  },
                  "tax":{
                     "component":"Magento_Tax\/js\/view\/checkout\/cart\/totals\/tax",
                     "config":{
                        "template":"Magento_Tax\/checkout\/cart\/totals\/tax",
                        "title":"Tax"
                     },
                     "sortOrder":"40"
                  },
                  "weee":{
                     "component":"Magento_Weee\/js\/view\/cart\/totals\/weee",
                     "config":{
                        "title":"FPT"
                     },
                     "sortOrder":"35"
                  }
               }
            }
         }
      }
   }
}
```

**Comment by krishprakash** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @krishprakash. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @krishprakash, here is your [Magento instance](https://i-22380-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-22380-2-3-develop.instances.magento-community.engineering/admin](https://i-22380-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @krishprakash 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-99237` were created

**Issue Available**: @krishprakash, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @daniel-ifrim. Thank you for your report. 
The issue has been fixed in magento/magento2#22387 by @krnshah in 2.3-develop branch
Related commit(s):
 - [d43eeb84d6d7b973d628f61cfb0ff97b4e70cb07](https://github.com/magento/magento2/commit/d43eeb84d6d7b973d628f61cfb0ff97b4e70cb07)
 - [20f677f7babe89e175a54d40d851b7611c93b495](https://github.com/magento/magento2/commit/20f677f7babe89e175a54d40d851b7611c93b495)

The fix will be available with the upcoming 2.3.3 release.

**Comment by magento-engcom-team** [comment]
Hi @daniel-ifrim. Thank you for your report. 
The issue has been fixed in magento/magento2#23058 by @abrarpathan19 in 2.2-develop branch
Related commit(s):
 - [02f8aa6c6e2e5c0d63635e4c2869bf30030dd594](https://github.com/magento/magento2/commit/02f8aa6c6e2e5c0d63635e4c2869bf30030dd594)
 - [ead8e63801f664f4010c307cd6a6e37ea2c15e1a](https://github.com/magento/magento2/commit/ead8e63801f664f4010c307cd6a6e37ea2c15e1a)

The fix will be available with the upcoming 2.2.10 release.
